src/modules/checkbox/checkbox.spec.ts

```diff
@@ -1,9 +1,9 @@
-describe('Semantic-UI: Components - smCheckbox', function() {
-  'use strict';
+import { smCheckboxModule } from './checkbox';
 
-  var $scope, $compile, $rootScope;
+describe('Semantic-UI: Components - smCheckbox', () => {
+  let $scope, $compile, $rootScope;
 
-  beforeEach(module('semantic.ui.components.checkbox'));
+  beforeEach(angular.mock.module(smCheckboxModule.name));
 
   beforeEach(inject(function(_$rootScope_, _$compile_) {
```

Comment:
please change this part to follow the same style as in [here](https://github.com/Semantic-Org/Semantic-UI-Angular/blob/master/src/modules/rating/rating.spec.ts#L13-L15)